From b1f945385550cdcec4d12cc0adfdaca0d107a796 Mon Sep 17 00:00:00 2001
From: Kevin Gaudet <kevin.gaudet@ihu-strasbourg.eu>
Date: Tue, 25 Sep 2018 11:50:30 +0200
Subject: [PATCH] Main: prevent compositor instances from having empty render
 targets

---
 OgreMain/src/OgreCompositorInstance.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/OgreMain/src/OgreCompositorInstance.cpp b/OgreMain/src/OgreCompositorInstance.cpp
index d8862f9498..22cb617f26 100644
--- a/OgreMain/src/OgreCompositorInstance.cpp
+++ b/OgreMain/src/OgreCompositorInstance.cpp
@@ -645,11 +645,17 @@ void CompositorInstance::createResources(bool forResizeOnly)
             deriveTextureRenderTargetOptions(def->name, &hwGamma, &fsaa, &fsaaHint);
             
             if(width == 0)
+            {
                 width = static_cast<size_t>(
                                             static_cast<float>(mChain->getViewport()->getActualWidth()) * def->widthFactor);
+                width = width == 0 ? 1 : width;
+            }
             if(height == 0)
+            {
                 height = static_cast<size_t>(
                                              static_cast<float>(mChain->getViewport()->getActualHeight()) * def->heightFactor);
+                height = height == 0 ? 1 : height;
+            }
             
             // determine options as a combination of selected options and possible options
             if (!def->fsaa)
-- 
2.17.1.windows.2

