diff -uNr a/Source/igtlutil/CMakeLists.txt b/Source/igtlutil/CMakeLists.txt
--- a/Source/igtlutil/CMakeLists.txt	2018-06-04 10:33:03.113929994 +0200
+++ b/Source/igtlutil/CMakeLists.txt	2018-06-04 10:33:55.361928135 +0200
@@ -77,6 +77,12 @@
   SOVERSION ${OpenIGTLink_VERSION_MAJOR}
 )
 
+if (ANDROID)
+	target_link_libraries(igtlutil
+				m
+	)
+endif()
+
 #TARGET_LINK_LIBRARIES(igtlutil
 #  ${LIBS}
 #  )
